app/src/main/res/drawable/thumb_code42.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:height="200dp"
+    android:viewportHeight="240"
+    android:viewportWidth="240"
+    android:width="200dp">
+    <path
+        android:fillColor="#fe5000"
+        android:pathData="M122.6,149.868c-16.6,0 -30,-13.5 -30,-30.1s13.5,-30 30.1,-30c13.7,0 25.6,9.3 29,22.6h31v-52.5H62.5v120h120.2v-52.6h-31c-3.5,13.3 -15.4,22.5 -29.1,22.6"/>
```

Comment:
use this vector drawable:

```
<vector xmlns:android="http://schemas.android.com/apk/res/android"
    android:width="120dp"
    android:height="120dp"
    android:viewportWidth="120"
    android:viewportHeight="120">
  <path
      android:fillColor="#fe5000"
      android:pathData="m60.1,90c-16.6,0 -30,-13.5 -30,-30.1 0,-16.6 13.5,-30 30.1,-30 13.7,0 25.6,9.3 29,22.6h31V0H0V120H120.2V67.4h-31C85.7,80.7 73.8,89.9 60.1,90"/>
</vector>

```

![Screenshot from 2021-03-12 22-12-03](https://user-images.githubusercontent.com/856916/110999040-0d78b880-8380-11eb-8d82-3510e46c50b8.png)

Reply:
Didn't see this, sorry. Updated.